lib/cmd/common/local.js

```diff
@@ -452,17 +387,17 @@ function resolveFileName(uri, options, cb) {
     },
     function (input, callback) {
       var result = input;
-      if (!result) { // if already found, don't need to try this
-        if (startsWith(uri, "/.fhclocal/")) {
-          result = path.join(".", uri);
-          log.silly('Container file requested: ');
-        }
+      // if already found, don't need to try this
+      if (!result && startsWith(uri, "/.fhclocal/")) {
+        result = path.join(".", uri);
+        log.silly('Container file requested: ');
       }
       return callback(undefined, result);
     },
     function (input, callback) {
       var result = input;
-      if (result) { // if already found, don't need to try this
+      // if already found, don't need to try this
+      if (result) {
         return callback(undefined, result);
```

Comment:
should be strict rule in passing undefined or null. I see you use both cb(undefined, result) and cb(null, result)